From 2302f9d8ba5bd3b5ec4d790d208840a3a2186370 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Fri, 20 Mar 2026 20:24:54 +0100
Subject: [PATCH 5/8] ia64: Add open_tree_attr() syscall

open_tree_attr() was added after ia64 was removed from the kernel.

Fixes: c4a16820d901 ("fs: add open_tree_attr()")
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 88b694ac4540..c0a31d05b10b 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -387,3 +387,4 @@
 464	common	getxattrat			sys_getxattrat
 465	common	listxattrat			sys_listxattrat
 466	common	removexattrat			sys_removexattrat
+467	common	open_tree_attr			sys_open_tree_attr
-- 
2.52.0

